src/Rangeslider.js

```diff
@@ -106,9 +106,12 @@ class Slider extends Component {
    * @return {void}
    */
   handleStart = e => {
-    const { onChangeStart } = this.props
-    document.addEventListener('mousemove', this.handleDrag)
-    document.addEventListener('mouseup', this.handleEnd)
+    const { onChangeStart } = this.props;
+    document.addEventListener('mousemove', this.handleDrag);
+    document.addEventListener('mouseup', this.handleEnd);
+    document.addEventListener('touchmove', _this.handleDrag);
```

Comment:
I think you meant `this` and not `_this`?

Reply:
Tis done